src/store/store.ts

```diff
@@ -575,11 +580,14 @@ export const useStore = defineStore('store', {
     /** Whether all the amalgamation (regular) steps are valid. */
     isAmalgamationRegularValid (): boolean {
       // *** TODO: add checks for review page components
+      const isFolioNumberValid = this.getFolioNumberValid
+
       return (
         this.getAmalgamatingBusinessesValid &&
         this.isDefineCompanyValid &&
         this.isAddPeopleAndRolesValid &&
-        this.isCreateShareStructureValid
+        this.isCreateShareStructureValid &&
+        isFolioNumberValid
```

Comment:
Since you're not doing anything else with this local variable, I recommend just using the getter here.

Reply:
Looks good. Resolving.